servicetalk-dns-discovery-netty/src/main/java/io/servicetalk/dns/discovery/netty/DefaultDnsServiceDiscovererBuilder.java

```diff
@@ -92,6 +94,20 @@ public DefaultDnsServiceDiscovererBuilder optResourceEnabled(final boolean optRe
         return this;
     }
 
+    /**
+     * Set the capacity of the datagram packet buffer (in bytes).
+     *
+     * @param datagramBufferCapacity the capacity of the datagram packet buffer
+     * @return {@code this}.
+     */
+    public DefaultDnsServiceDiscovererBuilder datagramBufferCapacity(final int datagramBufferCapacity) {
```

Comment:
`datagramBufferCapacity` -> `maxUdpPayloadSize`?

also consider clarifying in the comments (e.g. if DNS response exceeds this amount the UDP request will)